pjmedia/src/pjmedia/stream.c

```diff
@@ -1797,6 +1797,19 @@ static void handle_incoming_dtmf( pjmedia_stream *stream,
     event_duration = pj_ntohs(event->duration);
     is_event_end = (event->e_vol & PJMEDIA_RTP_DTMF_EVENT_END_MASK) != 0;
 
+    /* Correct duration and timestamp for codecs where clock_rate
+     * differs from sample rate
+     */
+    float ts_modifier = 1.0;
```

Comment:
Thank you. Only one small modification required. The variable declaration needs to be at the beginning of the block.